.github/workflows/stale.yaml

```diff
@@ -9,32 +9,39 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       issues: write
+      pull-requests: write
     steps:
-      - name: 'Mark stale issues'
+      - name: 'Mark stale issues and PRs'
         uses: actions/stale@v10
         with:
           days-before-issue-stale: 90
-          days-before-pr-stale: -1
+          days-before-pr-stale: 21
           days-before-close: -1
           stale-issue-label: 'lifecycle/stale'
           exempt-issue-labels: 'lifecycle/rotten'
           stale-issue-message: 'This issue is marked as stale after 90d of inactivity. After an additional 30d of inactivity, it will be closed. To prevent this issue from being closed, add a comment or remove the `lifecycle/stale` label.'
+          stale-pr-label: 'lifecycle/stale'
+          exempt-pr-labels: 'lifecycle/rotten'
+          stale-pr-message: 'This PR is marked as stale after 21d of inactivity. After an additional 14d of inactivity, it will be closed. To prevent this PR from being closed, add a comment or remove the `lifecycle/stale` label.'
 
-      - name: 'Mark rotten issues'
+      - name: 'Mark items rotten'
         uses: actions/stale@v10
         with:
-          days-before-issue-stale: 30
-          days-before-pr-stale: -1
+          days-before-issue-stale: 15
+          days-before-pr-stale: 7
```

Comment:
make the numbers and the text message consistent.

Reply:
@nirrozenbaum my understanding is:
no activity x days -> `stale` -> nothing for y days more -> `rotten` -> nothing for z days more -> `close`.

so message on move to `stale` should be that isssue/PR will close after (y+z) additional days of inactivity.
Hence (e.g.) `stale`->`rotten` after `7d` and `rotten`->`close` after `7d` more, for a total of 14d/2w.

If this is correct, the original message was wrong.
That's not to say I could have a typo in there as well, but is my understanding correct?

Reply:
I think the message is correct. It is saying two things:
1. Your issue/PR has been inactive for 90/21 days
2. In an additional 30.14 days the issue/PR will be closed

@nirrozenbaum @elevran

Reply:
sorry, confused. my bad.
the current messages are correct.